deps/googletest/cmake/internal_utils.cmake

```diff
@@ -133,7 +133,11 @@ macro(config_compiler_and_linker)
   set(cxx_exception "${cxx_base_flags} ${cxx_exception_flags}")
   set(cxx_no_exception
     "${CMAKE_CXX_FLAGS} ${cxx_base_flags} ${cxx_no_exception_flags}")
+  if(NOT WIN32)
   set(cxx_default "${cxx_exception}")
+  else()
+  set(cxx_default "${cxx_exception} /MD")
```

Comment:
As far as I know DPC++ compiler defines /MD by default, do we really need to provide it explicitly here?

Reply:
I haven't found any mentions of /MD or /MT in ninja.build of test part, but without that explicit /MD I see the error that gtest-all is built as a static one

Reply:
Oh, I see, so we use the same set of options for oneMKL build and for gtests, that use not DPC++ compiler. We will need to improve it in the future, but for now I agree, this is the only one solution.